README.md

```diff
@@ -142,6 +142,21 @@ An example of such a link would be:
 https://studio.opencast.org/?opencast.serverUrl=https://develop.opencast.org&upload.workflowId=fast&upload.seriesId=3fe9ea49-a671-4d1e-9669-0c96ff0f8f79
 ```
 
+In some situations, some part of the system modifies URLs and may corrupt GET parameters. For example, sometimes special case characters are changed or escaped. In these situations, you can encode your whole configuration as a JSON string, then encode it as hex string and pass it with the `config=` GET parameter. For example:
```

Comment:
This sounds like Studio would modify them in some situations.

```suggestion
You can also include your configuration in a JSON object, then encode it as hex string and pass it with the `config=` GET parameter. This might help to avoid problems with other systems stripping multiple parameters. For example:
```

Reply:
What do you think about this?

> You can also include your configuration in a JSON object, then encode it as hex string and pass it with the `config=` GET parameter. This might help to avoid problems if URLs (and thus the GET parameters) are processed (e.g. by an LMS) in a way that modifies special characters. For example:

This is basically your suggestion but replacing the "multiple parameters" part of the sentence.